frontend/src/app/pages/category-page/category-page.component.scss

```diff
@@ -4,7 +4,13 @@
 
 .title-row {
     margin-top: 3em;
-    font-size: 1.5em;
+    font-size: 2.5em;
+}
+
+#failureIcon {
```

Comment:
Mix of snakeCase with kebab-case (https://en.wikipedia.org/wiki/Letter_case#Special_case_styles)